components/gnome-connection-manager/gnome-connection-manager.p5m

```diff
@@ -0,0 +1,54 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2014 Alexander Pyhalov. All rights reserved.
+# Copyright (c) 2015 Jim Klimov. All rights reserved.
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+# Add-on for this package
+file path=usr/bin/gcm
+# Files delivered by source tarball, as of version 1.1.0, listed via:
+#   find . -type f | sed 's,^./,file path=usr/share/gnome-connection-manager/,'
+file path=usr/share/gnome-connection-manager/donate.gif
+file path=usr/share/gnome-connection-manager/gnome_connection_manager.py
```

Comment:
You don't set execute bit on the file. This would allow pkgdepend to find dependencies.